mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -71,6 +72,9 @@ class SetpointRawPlugin : public plugin::PluginBase,
 	ros::Subscriber local_sub, global_sub, attitude_sub;
 	ros::Publisher target_local_pub, target_global_pub, target_attitude_pub;
 
+	double thrust_scaling;
+	bool ignore_thrust_message_;
+
```

Comment:
That variables is not used outside of handle func, no reason to make them private members.

Reply:
Done.